mariadb-11.5.yaml

```diff
@@ -232,3 +233,42 @@ update:
     strip-prefix: mariadb-
     use-tag: true
     tag-filter: mariadb-11.5.
+
+test:
+  environment:
+    contents:
+      packages:
+        - mariadb-dev
+        - mariadb
+        - mysql-client
+        - shadow
+        - sudo-rs
+    environment:
+      DBDATA: /var/lib/mysql
+      DBUSER: mysql
+  pipeline:
+    - name: Check versions
+      runs: |
+        mysql --version
+        mariadb --version
+        mariadbd --version
+        mysqladmin --version
+        mariadb-admin --version
+    - name: Test mariadbd start and stop
+      uses: test/daemon-check-output
+      with:
+        setup: |
+          useradd $DBUSER
```

Comment:
if this is something required maybe we could use post-install scriplet for mariadb-client? does that make sense?

Reply:
Actually its just for testing, we shouldn't need this in the image since there will be  `accts` set.